libs/sdk-ui-dashboard/src/model/store/widgetDrills/widgetDrillSelectors.ts

```diff
@@ -198,72 +197,68 @@ export const selectInsightWidgetImplicitDrillToUrlByRef = createMemoizedSelector
 /**
  * @internal
  */
-export const selectInsightWidgetImplicitDrillsByRef = createMemoizedSelector((ref: ObjRef) =>
+export const selectConfiguredDrillsByWidgetRef = createMemoizedSelector((ref: ObjRef) =>
```

Comment:
Why was the `Implicit` part removed here?

Reply:
Because this not implicit drills this  drill come from widget.drills its configured drills. I want to distinguish between real implicit (generated base metadata) and configured drills by user. Whole selectors where flooded by wording IMPLICIT and was problem get know what does it mean.

Reply:
Ok, makes sense :)